acra/configuring-maintaining/general-configuration/acra-webconfig.md

```diff
@@ -0,0 +1,108 @@
+---
+title: acra-webconfig
+bookCollapseSection: true
+---
+
+# acra-webconfig
+
+`acra-webconfig` is simple web application that provide web UI for AcraServer's runtime configuration. Currently, it is
+under development and for now allows configuring 9 parameters of AcraServer:
+* `db_host`
+* `db_port`
+* `incoming_connection_api_port`
+* `d`
+* `poison_run_script_file`
+* `poison_shutdown_enable`
+* `zonemode_enable`
+
+{{< hint info >}}
+Parameter's description you can find on AcraServer's [command line flags]({{< ref "/acra/configuring-maintaining/general-configuration/acra_server.md#command-line-flags" >}}) page.
+{{< /hint >}}
+
+## Command line flags
+
+* `--destination_host`
+  IP or domain to AcraServer/AcraConnector daemon
+  Default is `localhost`
+
+* `--destination_port`
+  Port of AcraServer's HTTP API port or AcraConnector port that proxy API requests to AcraServer
+  Default is `9191`
+
+* `--incoming_connection_host`
+
+  Host for AcraWebconfig HTTP endpoint
+  Default is `127.0.0.1`
+
+* `--incoming_connection_port`
+
+  Port for AcraWebconfig HTTP endpoint
+  Default is `8000`
+
+* `--http_auth_mode`
+
+  Mode for basic auth. Possible values:
+    * `auth_on` (default): turn on basic auth
+    * `auth_off_local`: turn off basic auth if AcraServer/AcraConnector running on same host and accessible via `localhost` or `127.0.0.1`
+    * `auth_off`: turn off basic auth
+
+* `--static_path`
+
+  Path to static content
+  Default is `cmd/acra-webconfig/static`
+
+### Configuration files
+
+* `--config_file=<filename>`
+
+  Path to YAML configuration file.
+
+* `--dump_config`
+
+  Dump configuration to `configs/acra-server.yaml`.
+
+* `--generate_markdown_args_table`
+
+  Generate markdown file with text description of all flags.
+  Output file is `configs/markdown_acra-server.md`.
+  Works in pair with `--dump_config`.
+
+### Logging
+
+* `-d`
+
+  Log to stderr all `DEBUG`, `INFO`, `WARNING` and `ERROR` logs.
```

Comment:
- Based on the source code, it's more like this?

```suggestion
* `-d`

  Log to stderr only `DEBUG` logs and above.
  Do not produce `VERBOSE` logs.
```

Reply:
actually not. it turns on logs with debug and higher levels - https://github.com/sirupsen/logrus/blob/bdc0db8ead3853c56b7cd1ac2ba4e11b47d7da6b/logger.go#L374
So it will log debug, info, warning and error logs except trace level logs.